### PR TITLE
Receive message confirmations rebased

### PR DESCRIPTION
**Pull request by theworkerant**
I was looking at https://github.com/BlakeWilliams/Elixir-Slack/pull/64 and it seems it probably hasn't been merged because its out of date the file structure changes on current master. This should get you up to date— I've tried this out on my own bot and it seems to work fine.

### CODE REVIEW

File: lib/slack/bot.ex [hunk 1]
```diff
@@ -110,26 +110,24 @@ defmodule Slack.Bot do
 
   @doc false
   def websocket_handle({:text, message}, _conn, %{slack: slack, process_state: process_state, bot_handler: bot_handler} = state) do
-    message = prepare_message message
+    case prepare_message message do
```
Comment (theworkerant) [inline lib/slack/bot.ex:113]: I was a little unsure of these changes, but they seem to jive with current master.

**Comment by christhekeele** [comment]
Hey, @theworkerant – this looks good, but could you issue the PR to `christhekeele:receive-message-confirmations` instead? That should automatically update the open BlakeWilliams#64 PR.

**Comment by theworkerant** [comment]
Can't merge that unfortunately: https://github.com/christhekeele/Elixir-Slack/compare/receive-message-confirmations...theworkerant:receive-message-confirmations-rebased

**Comment by christhekeele** [comment]
Messy. I'll rebase everything together onto the PR branch such that you still are attributed for those changes. :+1:

**Comment by christhekeele** [comment]
Your code is now a part of my [receive-message-confirmations](https://github.com/christhekeele/Elixir-Slack/commits/receive-message-confirmations) branch, so the BlakeWilliams#64 PR now contains it. However, [it looks like the tests failed](https://travis-ci.org/BlakeWilliams/Elixir-Slack/builds/190054758)–if you fix them, and issue fixes to that branch, I'll happily keep sending them upstream to the PR until it reaches a good state.

Let me know if you want any assistance with fixing the tests.

**Comment by theworkerant** [comment]
Sure thing. I can probably fix them.

**Comment by theworkerant** [comment]
Here we go: https://github.com/theworkerant/Elixir-Slack/commit/fc800812b21e7a3177cddeb10094d9edfa038a56

**Comment by christhekeele** [comment]
PR updated. I have also made my `master` up to date with the `receive-message-confirmations` branch, in case you wish to just use that in your `mix.exs`.

**Comment by theworkerant** [comment]
Super! Thanks.

**Pull request merged**